pkg/archive/archive.go

```diff
@@ -1019,14 +1022,27 @@ func untarHandler(tarArchive io.Reader, dest string, options *TarOptions, decomp
 // If either Tar or Untar fails, TarUntar aborts and returns the error.
 func (archiver *Archiver) TarUntar(src, dst string) error {
 	logrus.Debugf("TarUntar(%s %s)", src, dst)
-	archive, err := TarWithOptions(src, &TarOptions{Compression: Uncompressed})
+	tarMappings := archiver.TarIDMappings
+	if tarMappings == nil {
+		tarMappings = &idtools.IDMappings{}
+	}
+	options := &TarOptions{
+		UIDMaps: tarMappings.UIDs(),
+		GIDMaps: tarMappings.GIDs(),
+	}
+	archive, err := TarWithOptions(src, options)
```

Comment:
This drops `Compression: Uncompressed`. Is that intentional?

Reply:
> `Uncompressed` is the default value

Ah, never mind them; ISTR some other part of the package used to default to compression, and I thought this code change changed behavior.

If the behavior doesn’t change, I don’t have any preference WRT making the decision explicit vs. making the code shorter.